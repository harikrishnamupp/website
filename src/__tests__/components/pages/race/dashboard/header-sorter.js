import React from 'react'
import renderer from 'react-test-renderer'
import HeaderSorter from '~components/pages/race/dashboard/header-sorter'

describe('Components : Race : Dashboard : Header Sorter', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<HeaderSorter stateName="Arizona" stateReports="race" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
