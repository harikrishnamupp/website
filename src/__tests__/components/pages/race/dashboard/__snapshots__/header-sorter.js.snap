// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Race : Dashboard : Header Sorter renders correctly 1`] = `
<div
  className="container"
>
  <p>
    The following tables reflect only those cases and deaths where
     
    race
     is known and reported by 
    Arizona
    . If this stateâ€™s reporting percentages are low, interpret with caution.
  </p>
</div>
`;
